frontend/src/metabase/collections/containers/CollectionContent.jsx

```diff
@@ -363,20 +343,75 @@ function CollectionContent({
   );
 }
 
-export default _.compose(
-  Bookmark.loadList(),
-  Databases.loadList(),
-  Collection.loadList({
-    query: {
-      tree: true,
-      "exclude-other-user-collections": true,
-      "exclude-archived": true,
+export function CollectionContent({ collectionId }) {
```

Comment:
!nit / maybe not worth doing

this is a huge file, you could split out the CollectionContentView to its own .jsx file and turn this one into .tsx.

Reply:
good idea! I did this!